docs/Containers/Container-Tasks.md

```diff
@@ -118,6 +118,52 @@ The following is the process by which you can create a usable image from your cu
     Evergreen YAML must also use a **&lt;SHA&gt;** tag, which would be the hash
     of the corresponding commit in our [image repository](https://github.com/evergreen-ci/container-initial-offering-dockerfiles).
 
+The following is a starter Dockerfile that abides by our image policy and
```

Comment:
Can we also link to the image policy here? I know it's already listed above, but some users may be tempted to skip to the example template.